docs/tutorials/io.rst

```diff
@@ -35,26 +42,39 @@ touched. So, you can tickle your device to make it laugh like this::
         else:
             display.show(Image.SAD)
 
-With one hand, hold your device by the GND pin. Then, with your other hand,
+With one hand, hold your micro:bit by the GND pin. Then, with your other hand,
 touch (or tickle) the 0 (zero) pin. You should see the display change from
 grumpy to happy!
 
+When you use the latest micro:bit **V2** you can also change the default
+behaviour of the pin, so that you don't have to touch GND at all.
```

Comment:
Should we include a code snippet here to indicate how?

Reply:
added in https://github.com/bbcmicrobit/micropython/pull/709/commits/874bd69766805c7075088acd6b683e601ac82d1d